### PR TITLE
refactor(web-app): Improve the name of a test asset

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: plugins/reporters/web-app/src/funTest/kotlin/WebAppReporterFunTest.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ import org.ossreviewtoolkit.utils.test.readOrtResult
 class WebAppReporterFunTest : WordSpec({
     "WebAppReporter" should {
         "successfully export to a web application" {
-            val ortResult = readOrtResult(getAssetFile("dummy-expected-output-for-analyzer-result.yml"))
+            val ortResult = readOrtResult(getAssetFile("input-analyzer-result.yml"))
```
Comment (sschuberth) [inline plugins/reporters/web-app/src/funTest/kotlin/WebAppReporterFunTest.kt:34]: As this file also contains a scan result, I propose to instead use e.g. "scan-result-for-synthetic-gradle-lib.yml".

File: plugins/reporters/web-app/src/funTest/kotlin/WebAppReporterFunTest.kt [hunk 2]
```diff
@@ -31,7 +31,7 @@ import org.ossreviewtoolkit.utils.test.readOrtResult
 class WebAppReporterFunTest : WordSpec({
     "WebAppReporter" should {
         "successfully export to a web application" {
-            val ortResult = readOrtResult(getAssetFile("dummy-expected-output-for-analyzer-result.yml"))
+            val ortResult = readOrtResult(getAssetFile("scan-result-for-synthetic-gradle-lib.yml.yml"))
```
Comment (sschuberth) [inline plugins/reporters/web-app/src/funTest/kotlin/WebAppReporterFunTest.kt:34]: There's a double `.yml` extension now.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Comment by sschuberth** [comment]
The pattern in `plugins/reporters/web-app/src/funTest/assets/.gitattributes` now does not match anymore, but actually I believe the whole file could be dropped, as the file is only used for deserialization, and not for comparison / as an expected result.

**Comment by fviernau** [comment]
> but actually I believe the whole file could be dropped,

What would you use as a replacement?

**Comment by sschuberth** [comment]
> > but actually I believe the whole file could be dropped,
> 
> What would you use as a replacement?

Nothing, the `.gitattributes` file should simply be removed. I don't think the file is needed at all, and I made a mistake of putting it there in the context of factoring out the reporters.

**Comment by sschuberth** [comment]
> The pattern in `plugins/reporters/web-app/src/funTest/assets/.gitattributes` now does not match anymore, but actually I believe the whole file could be dropped, as the file is only used for deserialization, and not for comparison / as an expected result.

Mind adding the rationale for being able to remove the `.gitattributes` file to the commit message? At your choice, the removal could also happen in a preceding commit, as the file could have been removed independently already.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**